script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1B.txt

```diff
@@ -134,11 +134,11 @@ She smiles, but her eyes are fixed on my face like a sticky thread, as they have
 Hisui intervenes.
 Without reading the atmosphere... no, without being affected by the atmosphere, Hisui neutralized the tension created by Arach.//場の空気を読まない……いや、場の空気に左右されない翡翠の態度が、阿良句女史のテンションを中和してくれたようだ。In this line and line 139, there's a 女史 suffix for Arach which is usually translated as Miss, Ms, Mrs. I think we can leave it as it is, just pointing it out in case we want to change it.
 <Page38>
-"Oh, Maid-chan really now. You really are the epitome of a maid."
+"Oh, Maid-chan, really now. You really are the epitome of a maid."
```

Comment:
> もう。メイドチャンったら、ホント、メイドの鑑ねぇ

I think the 'really now' can be removed. If it feels empty than maybe it can be replaced with 'come on'